src/client.h

```diff
@@ -554,10 +557,6 @@ class Client : public con::PeerHandler, public InventoryManager, public IGameDef
 	// And relations to objects
 	std::unordered_map<int, u16> m_sounds_to_objects;
 
-	// HUD
-	// Mapping from server hud ids to internal hud ids
-	std::unordered_map<u32, u32> m_hud_server_to_client;
```

Comment:
please add ref getter instead of make it public

Reply:
sure